src/components/new-request/search.vue

```diff
@@ -645,6 +675,10 @@ export default class Search extends Mixins(CommonMixin) {
 .mobile-btn {
   width: 17rem !important;
 }
+.selected-item {
+  margin-top: 10px !important;
+  margin-bottom: 5px !important;
+}
```

Comment:
Can you add a screenshot showing what this change does please?

Reply:
This is not used. I added it while playing with the 'Select action' input text to show 2 lines. I will remove this.

Reply:
Fixed.